integration-tests/plugins/__tests__/cart/store/ff-medusa-v2.ts

```diff
@@ -0,0 +1,95 @@
+import { Region } from "@medusajs/medusa"
```

Comment:
thought: I wonder if it would make sense to create dedicated integration test folders for Medusa V2, so we have those centralised instead of spread out in different subfolders?

cc @adrien2p

Reply:
Yes definately, we started that a bit when we created the workflows but we could do that for everything indeed, do we have a lot of them spread out there? 😂

Reply:
Yes, I agree. Does it make sense to do this refactor in a separate ticket if we have these in multiple places?

Reply:
Yeah sure, fine to do it in a follow-up PR